model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -86,18 +86,28 @@ class LicenseFindingDeserializer : StdDeserializer<LicenseFinding>(LicenseFindin
         return when {
             node.isTextual -> LicenseFinding(node.textValueOrEmpty(), sortedSetOf(), sortedSetOf())
```

Comment:
Not changed in this commit, but also applies here: If we already know the node to be textual, I believe it's already non-null, so we should be able to just use `textValue()` here.

Reply:
Done.